src/parthenon_manager.cpp

```diff
@@ -222,6 +222,9 @@ void ParthenonManager::RestartPackages(Mesh &rm, RestartReader &resfile) {
   size_t nCells = bsize[0] * bsize[1] * bsize[2];
 
   // Get list of variables, assumed same for all blocks
+
+  // TODO(JL) this doesn't work anymore, will update this in sparse update
+
```

Comment:
How much is broken? Just when using sparse variables?

Reply:
Nothing is broken right now, even when using current sparse variables. But with the sparse update, you will be able to write true sparse variables in a restart file, but won't be able to read them back correctly. So this needs to be updated in that PR, but it shouldn't make a difference as long as we don't have true sparse variables yet. I'll change this to "this won't work" instead of "doesn't work anymore".